src/crepe/api/Sprite.cpp

```diff
@@ -15,7 +15,8 @@ Sprite::Sprite(game_object_id_t id, const shared_ptr<Texture> image, const Color
 	: Component(id),
 	  color(color),
 	  flip(flip),
-	  sprite_image(image) {
+	  sprite_image(image),
+	  aspect_ratio(sprite_image->get_width() / sprite_image->get_height()) {
```

Comment:
Dit gaat niet werken. `sprite_image->get_width()` returned (volgens mij) een int en `sprite_image->get_height()` returned (volgens mij) ook een int. Je moet hier dus casten naar double (of float) zodat de uitkomst niet afgerond wordt.
PS Dit doet mij denken dat je überhaupt nog niet hebt getest zonder animator, dat zou ik wel verwachten.